src/engine/input/key_mapping/InputActionItem.tscn

```diff
@@ -1,9 +1,21 @@
-[gd_scene load_steps=4 format=2]
+[gd_scene load_steps=8 format=2]
 
 [ext_resource path="res://src/engine/input/key_mapping/InputActionItem.cs" type="Script" id=1]
 [ext_resource path="res://src/gui_common/thrive_theme.tres" type="Theme" id=2]
 [ext_resource path="res://src/gui_common/fonts/Lato-Regular-Smaller.tres" type="DynamicFont" id=3]
 
+[sub_resource type="StyleBoxEmpty" id=1]
```

Comment:
The plus sign does become white and it goes down slightly, I think that makes it clear that this is pressable. I honestly doubt that it will look good with the background on hover, but thats just my design taste.

Reply:
Okay, maybe just leave that then, I thought so just to make it look consistent with the other UIs.